pymetrics/instruments.py

```diff
@@ -21,7 +21,7 @@
 try:
     from typing import Literal  # type: ignore
 except ImportError:
-    from typing_extensions import Literal
+    from typing_extensions import Literal  # type: ignore
```

Comment:
This and other similar changes are to ignore newer mypy checks that were throwing errors only in python 3.8

Reply:
Same comment about filing tickets to fix this properly.